docs/source/tutorials/configuration.rst

```diff
@@ -135,76 +101,88 @@ open the file after it is rendered.
 
    $ manim <file.py> SceneName -o myscene -i -n 0,10 -c WHITE
 
+.. tip:: There are many more command line flags that manim accepts.  All the
+	 possible flags are shown by executing ``manim -h``.  A complete list
+	 of CLI flags is at the end of this document.
 
 
 The config files
 ****************
 
 As the last example shows, executing manim from the command line may involve
-using many flags at the same.  This may become a nuisance if you must execute
-the same file many times in a short time period, for example when making small
-incremental tweaks to your scene script.  For this purpose, manim can also be
-configured using a configuration file.
+using many flags at the same time.  This may become a nuisance if you must
+execute the same script many times in a short time period, for example when
+making small incremental tweaks to your scene script.  For this purpose, manim
+can also be configured using a configuration file.  A configuration file is a
+file ending with the suffix ``.cfg``.
 
 To use a configuration file when rendering your scene, you must create a file
-with name :code:`manim.cfg` in the same directory as your scene code.
+with name ``manim.cfg`` in the same directory as your scene code.
 
-.. warning:: The config file **must** be named :code:`manim.cfg`. Currently,
-             manim does not support config files with any other name.
+.. warning:: The config file **must** be named ``manim.cfg``. Currently, manim
+             does not support config files with any other name.
```

Comment:
is this even true? What about --config_file ?

Reply:
This is true in the sense that the user and folder files can only have that name. If you use --config_file, the file can have any name. I agree this is confusing. Open to suggestions on how to implement it. (The problem is that the user and folder files are loaded and parsed way before the CLI arguments are parsed. So there's currently no way of changing that without a *major* refactor...)